### PR TITLE
ncm-nss: wait for nscd to restart before returning

### PR DESCRIPTION
**Pull request by ned21**
This is a very simple change we'd like in 16.4. I have a another branch that replaces LC::Check with FileWriter and includes unit tests which I can push for 16.6.

### CODE REVIEW

File: ncm-nss/src/main/perl/nss.pm [hunk 1]
```diff
@@ -113,6 +113,10 @@ sub Configure {
             my $nscdrestart = CAF::Process->new(
                 ['service', 'nscd', 'condrestart'], log => $self);
             $nscdrestart->execute();
+	    # wait for nscd to restart before returning in case
```
Comment (stdweird) [inline ncm-nss/src/main/perl/nss.pm]: are you aware of https://github.com/quattor/configuration-modules-core/blob/master/ncm-authconfig/src/main/perl/authconfig.pm#L379? it's not condrestart, but for sure does not assume this a simple restart will just work.

**Comment by ned21** [comment]
Unit test failure seems unrelated to this change?

File: ncm-nss/src/main/perl/nss.pm (hunk 1 above)
Comment (ned21) [inline ncm-nss/src/main/perl/nss.pm]: I was not and we've not seen that particular problem.

**Comment by stdweird** [comment]
@ned21 @alvarosimon is on it. expect a PR soonish

**Comment by alvarosimon** [comment]
@ned21 @stdweird https://github.com/quattor/configuration-modules-core/pull/782 should fix ncm-opennebula issue

**Comment by stdweird** [comment]
retest this please

**Pull request merged**